<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Camera</class>
 <widget class="QDialog" name="Camera">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>196</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>196</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>196</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Viewerport configuration</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/icons/postprocess.png</normaloff>:/icons/postprocess.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Camera configuration</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="minimumSize">
           <size>
            <width>151</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>151</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Movement increment</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButton">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="icons.qrc">
            <normaloff>:/icons/move.png</normaloff>:/icons/move.png</iconset>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="moveInc">
          <property name="decimals">
           <number>2</number>
          </property>
          <property name="maximum">
           <double>99.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
          <property name="value">
           <double>0.100000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="minimumSize">
           <size>
            <width>151</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>151</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Rotation increment</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButton_2">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="icons.qrc">
            <normaloff>:/icons/rotation.png</normaloff>:/icons/rotation.png</iconset>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="rotateInc">
          <property name="maximum">
           <double>180.000000000000000</double>
          </property>
          <property name="value">
           <double>15.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="minimumSize">
           <size>
            <width>151</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>151</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Zoom increment</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButton_3">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="icons.qrc">
            <normaloff>:/icons/zoom.png</normaloff>:/icons/zoom.png</iconset>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="zoomInc">
          <property name="maximum">
           <double>99.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
          <property name="value">
           <double>1.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnReset">
       <property name="text">
        <string>Reset</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>15</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnOk">
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>15</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>10</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
